assets/package-lock.json

```diff
@@ -95,11 +95,11 @@
       }
     },
     "../deps/phoenix": {
-      "version": "1.6.16",
+      "version": "1.7.10",
       "license": "MIT"
     },
     "../deps/phoenix_html": {
-      "version": "3.3.1"
+      "version": "3.3.3"
```

Comment:
https://github.com/mbta/skate/pull/2300#discussion_r1419106151 Seems like we might want to merge the dep update before this?